Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -334,4 +341,18 @@ export class AssetManager {
 
     return await res.text();
   }
+
+  public async getNpmPackageVersions(packageName: string): Promise<string[]> {
+    const registryUrl = `https://registry.npmjs.org/${packageName}`;
+    const response = await fetch(registryUrl);
```

Comment:
How are you handling errors? I dont see a try catch wrapper at the calling site as well.

Reply:
The root caller has error handling, this is only called from `getProjTemplates` method which handles any thrown error. Its unlikely that this will be called elsewhere in the future but just in case I have added a try catch to the function itself so it isnt assumed that you need to add one to any future callers.